doc/manual/config-basic.rst

```diff
@@ -40,6 +40,17 @@ A jury or administrative user can also be associated with a team. This
 will enable that user to submit solutions to the system, or resubmit
 edited team solutions.
 
+Resetting the password for a user
+---------------------------------
+
+If you do not have access anymore to any admin user, you can use the following
+command to reset the password of a user to a random value::
+
+  webapp/bin/console domjudge:reset-user-password admin
+
+Replace `admin` with the username of the user you want to reset the password of.
+The password will be generated and displayed.
```

Comment:
I think \`admin\` is Markdown, and it should be \`\`admin\`\` for RST.

Aside, some alternative wording options:
"Replace \`\`admin\`\` with the username of the user you want to reset the password **for**." (More natural? Same in setArgument.)
"The new password will be displayed." (Hopefully makes it clearer that the password will be updated in the database, and not just generated.)